dspace/config/dspace.cfg

```diff
@@ -1,4 +1,4 @@
-#
+${}#
```

Comment:
No it isn't -- not sure why it doesn't cause an error during build. Thanks for catching that